test/cql-pytest/test_group_by.py

```diff
@@ -73,8 +106,6 @@ def test_group_by_clustering_prefix(cql, table1):
     assert [(0,0,0,1), (0,1,0,3)] == list(cql.execute(f'SELECT p,c1,c2,v FROM {table1} WHERE p=0 GROUP BY p,c1'))
 
 # Adding a LIMIT should be honored.
```

Comment:
Again, please don't remove the "Reproduces" comments - they are important!
Just yesterday we had a presentation from @xtrey who said this was one of the things he wanted to achieve with Allure (as you can see here, Allure isn't needed for tests to say what they reproduce in a comment).

Reply:
Unfortunately, this comment is not visible on the test results page, where it will provide more value.